services/src/main/resources/org/keycloak/protocol/oidc/endpoints/3p-cookies-step1.html

```diff
@@ -5,15 +5,19 @@
   </head>
   <body>
     <script type="module">
-      // Check if the browser has granted us access to 3rd-party storage (such as cookies).
-      const hasAccess = await hasStorageAccess();
+      attemptCookie();
```

Comment:
This function no longer exists, it should be calling `checkStorageAccess()` instead.
```suggestion
      checkStorageAccess();
```